### PR TITLE
Fix redirect issue on video player

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: src/controllers/playback/videoosd.js [hunk 1]
```diff
@@ -1341,7 +1341,7 @@ define(['playbackManager', 'dom', 'inputManager', 'datetime', 'itemHelper', 'med
                 });
             } catch (e) {
                 require(['appRouter'], function(appRouter) {
-                    appRouter.showDirect('/');
+                    window.location.href = 'index.html';
```
Comment (MrTimscampi) [inline src/controllers/playback/videoosd.js:1344]: You can use `appRouter.goHome` to do this (It's defined in site.js), it's less hacky than overriding the href.

Comment (dkanada) [reply src/controllers/playback/videoosd.js:1344]: I didn't want to mess with the horrendously long define statement at the top. We use this in a few other places, so I figured we could just replace them all at some point.

Comment (MrTimscampi) [reply src/controllers/playback/videoosd.js:1344]: You're already requiring appRouter right above, though ;)

It's already in there at that point, you only have to call it.

Comment (dkanada) [reply src/controllers/playback/videoosd.js:1344]: Well I feel stupid now :laughing: I was looking at the top for that dependency.

File: src/controllers/playback/videoosd.js [hunk 2]
```diff
@@ -1341,7 +1341,7 @@ define(['playbackManager', 'dom', 'inputManager', 'datetime', 'itemHelper', 'med
                 });
             } catch (e) {
                 require(['appRouter'], function(appRouter) {
-                    appRouter.showDirect('/');
+                    appRouter.goHome()
```
Comment (MrTimscampi) [inline src/controllers/playback/videoosd.js:1344]:
```suggestion
                    appRouter.goHome();
```

**Pull request merged**